x-pack/plugins/ml/server/routes/trained_models.ts

```diff
@@ -54,6 +57,45 @@ export function filterForEnabledFeatureModels<
   return filteredModels;
 }
 
+export const getInferenceServicesProvider = (client: IScopedClusterClient) => {
```

Comment:
I wonder is `getInferenceServicesProvider` and `getInferenceServices` are slightly misleading names as this doesn't return the services, but rather modifies a supplied list of models.
Maybe `populateInferenceServices` or `setInferenceServices` or something like that.

Reply:
renamed in https://github.com/elastic/kibana/pull/175880/commits/78ef95e7683611d1c5418c81e7de18507453f421